source/quick-start.txt

```diff
@@ -14,6 +14,52 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Complete the
+:atlas:`Get Started with Atlas </getting-started>` guide to set up a new
```

Comment:
Issue:

This linked guide actually currently shows everything that this guide shows, including running CRUD operations. Therefore, if you complete that guide, there's no reason to continue with this one.

Suggestion:
You might consider linking to this one instead for now since it ends with "Get Connection String": https://www.mongodb.com/docs/guides/atlas/account/